{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "masked": true,
          "val": 4
        },
        "etr": "",
        "start": "2020-12-07T20:50:45-0400"
      },
      "geom": {
        "p": [
          "w~arG`gxhK"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
